### ISSUE TITLE
Ways to turn off magnetization on the magnetic attractor unit

### ISSUE THREAD

**Issue by DripThanos** [mekanism/Mekanism-Feature-Requests#481]
### Describe the feature you'd like

I would like for there to be a way to disable magnetization without going to the module tweaker and manually setting it to "OFF". Therefor I have considered suggesting a new block that prevents you from magnetizing specified items (in a filter you set in the UI, similar to the logistical sorter and other items with filters) In this same UI, you can also adjust the range that magnetization is disabled in (can set X Y and Z separately), an upgrade and security menu (only accepts energy(1) and anchor upgrades), a toggle button that inverts the security setting's magnetization as well as redstone control. 

If security is set to "PRIVATE" It will only affect the owner's magnetization within the radius, If its set to "TRUSTED" it will only affect magnetization for you and everyone on your trusted list within the radius. Said magnetization for the security settings is affected by the filter as well as redstone control which will be discussed in the next paragraph. If the magnet invert button is on, all of the prior effects of the security setting will be reversed(2)

If redstone control is set to "NORMAL" and it receives no redstone signal, the block disables itself and allows you to pick up any item(s) specified in that filter and vice versa(3). 

I'm not sure if it should require power to operate, but if it needs it for any reason (balancing etc) the power required per tick should scale on the radius(4) and if it runs out of power, it ceases all function.

TLDR, its a more advanced version of Botania's Solegnolia or other mods that add a similar block for their respective magnets.

### Describe alternatives you've considered

If that block is not possible, here are some other solutions:
- ~~adding a separate keybind to disable the magnetic attraction unit~~ Isn't really needed.
- same thing as the block, but you set anchors instead through the module. Similar to Super Sound Muffler's anchors. (You can have an unlimited number of them however.)
- ~~allowing the magnetic attraction to be disabled using the Feet mode switch toggle; configurable in the module.~~ (already a thing)

### Additional context

(1) Energy upgrades will only be accepted if it is decided that the block requires power to operate
(2) It will allow everyone else but you to magnet items within the radius when set to "PRIVATE" for example.
(3) If redstone control is set to "INVERTED" and receives a redstone signal it will do the same as mentioned prior.
(4) example: The formula would be P*(X+Y+Z). P = power (per tick) X, Y, and Z = the values set in the range

**Comment by beholderface** [comment]
There's already a keybind that toggles the magnetic attraction unit (if handle mode switch is on in the module tweaker, iirc). Should be listed as "feet mode switch" in controls settings.

**Comment by DripThanos** [comment]
I'll edit my post then, but i really just want a way to turn it off in some areas. (ex. i pick up all of the crops in my create farms either breaking the farm or making it lose out on resources) without having to remember to turn it on when im building or mining.

**Comment by zyxkad** [comment]
I think it should block item in the range to be attracted, but not to disable the player's magnet in the range,
or if the magnet power will pass the area, it will be blocked
